### PR TITLE
fix: main entry for error free imports

### PR DESCRIPTION
**Pull request by Keyrxng**
Resolves https://github.com/ubiquibot/permit-generation/issues/41

This does prevent the usage of `../core/` or `../types/` but seeing as the main entry was `core` which includes everything anyway and was the previous default it seems like this is irrelevant?

QA:

image

### CODE REVIEW

**Review by gitcoindev — APPROVED** [review]
Setting up imports correctly can be tricky, thank you for fixing this.

**Comment by gentlementlegen** [comment]
Code looks fine, will test with a fresh project and approve if there is not issue.

**Review by gentlementlegen — APPROVED** [review]

**Comment by Keyrxng** [comment]
@gentlementlegen both the commit and PR title are correctly named as per our last convo about package publishing, so merging this should update and publish automatically and correctly right?

**Comment by gentlementlegen** [comment]
@Keyrxng It should be after we merge `development` into `main`.

**Pull request merged**
